### DISCUSSION TITLE
Problem with PHPMailer: error_email_not_sent

### DISCUSSION THREAD

**Discussion by nschuhm** [rubenarslan/formr.org#517] [Q&A] [answered]
Hi Ruben,

Since a few days we have problems sending emails to participants.

It is not with all participants, but with some: There is an error message "error_email_not_sent" in the user logs. See also screenshot.

We run our instance at the University of Münster. Daniela Feistauer told me to contact you here. Could you check what the problem might be and if it is related to the formR server?

We do not think that it has to do with the users' mail providers, since the error message appeared for at least 3 users at the same time (about 7 days ago).

Thank you very much.

Nils Schuhmacher
![Screenshot (1)](https://github.com/rubenarslan/formr.org/assets/158168831/51c3e032-e45d-4f88-9c50-e068b75e04e2)

**Comment by dfeistauer** [comment]
Hi Ruben, 

I have been experiencing the same issue for the past two weeks. I resolved it by returning the subject to the mail module, which then allowed the emails to be sent. However, this is not a ideal solution. Unfortunately, I do not have any screenshots and my study is now finished.

Daniela

**Comment by rubenarslan** [comment, marked as answer]
From the outside, I suspect problems with your local email server at the university rate limiting you. Your local IT should be able to help you rule that out or in.

**Comment by nschuhm** [comment]
Hi Ruben, thanks for your reply. I potentially resolved the issue. You were right: It was related to our email server. Best, Nils

**Comment by frako22** [comment]
Hi everyone,

We have a similar issue: According to formr, the e-mail and SMS were sent but my colleague received neither an e-mail nor a SMS. The same happened with the reminders. What could be the reason?

Thanks in adavance!

Best,
Franzi

**Comment by rubenarslan** [comment]
And you are using formr.org? Or formr.uni-muenster.de or some other local hosted machine?

**Reply by frako22** [reply]
Yes, formr.org. It just affects one person. I tried it with my own account and had no issues.

**Reply by rubenarslan** [reply]
Then it presumably has to do with the email server of that person, i.e. the emails are being silently blocked. Nothing we can do about it in formr, that person should use a different email provider or talk to theirs if it's a university one.

**Reply by frako22** [reply]
Yes, we found the problem. It is indeed an issue with clickatell and phone provider. The e-mail arrived though - sorry for the false alarm (we checked the wrong address :/). Thank you so much!
